### ISSUE TITLE
Sync Your Notes button should truncate, not wrap

### ISSUE THREAD

**Issue by ryanfeeley** [mozilla/notes#153]
<img width="184" alt="screenshot 2017-07-18 10 07 21" src="https://user-images.githubusercontent.com/68213/28321536-79dd7132-6ba1-11e7-8c8f-f3647ed5b802.png">

I tried to use the typical code:

```
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
```

But I don't think it works with flex-box. Ideas?

**Comment by ryanfeeley** [comment]
Not a stretch. Not a stretch!

**Comment by vladikoff** [comment]
@cedricium are you able to help with this?

**Comment by pdehaan** [comment]
@ryanfeeley How would truncation work? The button would say "Sync Your..."? Seems unfortunate loss of the key word in that phrase.

Or should the string just change the string to "Sync Notes" and we hope it never needs to wrap/truncate (which wouldn't help for translations)?

**Comment by cedricium** [comment]
@vladikoff I tried working on this issue when it was originally opened. The best I could do was get the button to not wrap but the text wouldn't truncate (no "..." appeared).

From the little research I did I found out buttons need some CSS magic in order to get the ellipsis to show. I'll see if I can get it working, but @pdehaan brings up a good point so I wonder what @ryanfeeley thinks.

**Comment by pdehaan** [comment]
I wonder if in addition to `white-space`, `overflow`, and `text-overflow`, if it needs an explicit container `width`: https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

**Comment by cedricium** [comment]
I think that along with doing [what's described here](https://stackoverflow.com/questions/9905409/is-it-possible-to-use-text-overflow-ellipsis-on-a-button-element) should work. I'll give this a shot and report back.

**Comment by Natim** [comment]
Released in 1.7.0 https://github.com/mozilla/notes/issues/188
